test/verify/machineslib.py

```diff
@@ -1445,6 +1465,12 @@ def fill(self):
                 b.click("#start-vm") # TODO: fix this, do not assume initial state of the checkbox
             # b.set_checked("#start-vm", self.start_vm)
 
+            if self.choose_storage:
+                b.wait_visible("#choose-storage")
+                b.click("#choose-storage") # TODO: fix this, do not assume initial state of the checkbox
```

Comment:
You can get the checkbox value from the testing framework. The TODO comments are probably outdated.
Check wait_checked, wait_not_checked functions.